<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Algo Tester</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ICON.png</normaloff>ICON.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="SelectCase">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Select a testcase</string>
    </property>
   </widget>
   <widget class="QLabel" name="UsingCase">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Using testcase file &lt;none&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DrawChart">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw test chart</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="ValidAlgos">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>200</width>
      <height>240</height>
     </rect>
    </property>
    <property name="title">
     <string>Select algos to test</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>412</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
